recipe/meta.yaml

```diff
@@ -76,7 +76,7 @@ outputs:
     script: install-win-wrapper.bat  # [win]
     requirements:
```

Comment:
Could you add?
```
build:
  skip: True  # [py<36]
```
see https://github.com/dmlc/xgboost/blob/eb69c6110ae991ffcf05b60940e25ff8eb134a28/python-package/setup.py#L345